pages/about.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>about page</title>
+    <link rel="stylesheet" href="../styles/about.css">
+    <link rel="stylesheet" href="../styles/index.css">
+    <script src="https://kit.fontawesome.com/3f00b2ab29.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header class="header-sect" >
+        
+        <div class="nav-bar">
+            <nav>
+                <button id="hide" class="hamburger" type="button"><i  id="bars" class="fa-solid fa-bars fa-2x icon-list"></i></button>
+                <ul class="mobile-menu" id="anchors">
+                    <li class="padding"><a class="pad" href="./index.html">Home</a></li>
+                    <li class="padding"><a class="pad" href="./about.html">About</a></li>
+                    <li class="padding"><a class="pad" href="#">Contact</a></li>
+                    <li class="padding"><a class="pad" href="#">Join</a></li>
+                  </ul>
+
+                <!-- desktop menu -->
+                <div class="desktop-menu">
+                    <img id="logo" src="../img/logo.jpg" alt="logo of company">
+                    <a href="./index.html">Home</a>
+                    <a href="./about.html">About</a>
+                    <a href="#">Program</a>
+                    <a href="#">Join</a>
+                    <a href="#">Sponsor</a>
+                    <a href="#">Campaign</a>
+
+                </div>
+                
+            </nav>
+        </div>
+            <div class="margin-header">
+                <h3 class="orange-color" id="header-text-1">
+                    "Hello people from all round the world"
+                </h3>
+                <h1 class="orange-color header-text2">GET INSPIRED BY THE BEST PEOPLE FROM AROUND THE WORLD ABOUT CLIMATE CHANGE</h1>
+            </div>
+            <p class="paragraph" id="paragraph-about">We are a team og people dedicated to fixing the global crisis on climate change. we hope weill join our cause to be the first generation to leave te planet better than we met it. A good heart is enough one person can make a differencce</p>
```

Comment:
- [ ] Please align the div with a margin-header class in the center of the page.
- [ ] Also the paragraph with the paragraph class align in the middle of the page.